ivy/functional/frontends/torch/tensor.py

```diff
@@ -373,6 +373,10 @@ def new_ones(self, size, *, dtype=None, device=None, requires_grad=False):
 
     def floor(self, *, out=None):
         return torch_frontend.floor(self)
+    
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
+    def not_equal(self, other, *, out=out):
```

Comment:
```suggestion
    def not_equal(self, other, *, out=None):
```

Reply:
@hello-fri-end  thank you for your input, I've applied the changes suggested above by changing ``` out=out ``` to ```out=None``` yet I wanted to double check if i should change the test to the following (since it has an alias) 
```
@handle_frontend_test(
    fn_tree="torch.ne",
    aliases=["torch.not_equal"],
    dtype_and_inputs=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"),
        num_arrays=2,
        allow_inf=False,
        shared_dtype=True,
    ),
)
def test_torch_not_equal(
    *,
    dtype_and_inputs,
    on_device,
    fn_tree,
    frontend,
    test_flags,
):
    input_dtype, inputs = dtype_and_inputs
    helpers.test_frontend_function(
        input_dtypes=input_dtype,
        frontend=frontend,
        test_flags=test_flags,
        fn_tree=fn_tree,
        on_device=on_device,
        input=inputs[0],
        other=inputs[1],
    )
    ```

Reply:
Hey @mustafaa7med, you should not use `handle_frontend_test` decorator for instance methods. Currently, `handle_frontend_method` does not have an `aliases` parameter but it's definitely in our TODO list. For now, just adding `ne = not_equal` alias in the `tensor.py` file should suffice. The test that you have written currently fails on the CI with an error saying `on_device` not found. Could you please fix this? (https://github.com/unifyai/ivy/actions/runs/5040147031/jobs/9038722382?pr=15701#step:3:431)